tests/integration/test_uninstall.py

```diff
@@ -243,3 +243,54 @@ def test_uninstall_multiple_categories(pipenv_instance_private_pypi):
 
         assert "six" not in p.lockfile.get("prereq", {})
         assert "six" not in p.lockfile["default"]
+
+
+@pytest.mark.uninstall
```

Comment:
I see some of the uninstall tests are marked with `@pytest.mark.install`, and some are not. Is this based on whether the test calls `install` during the test? Something else? Let me know if you want me to mark these two tests with `@pytest.mark.install` also.

Reply:
>  Is this based on whether the test calls install during the test? 
Yeah, the test markers needs some separate TLC -- but that is the idea.

Reply:
Thank you. I have marked these tests with `@pytest.mark.install`.

I saw there are some other tests in `test_uninstall.py` that call `install`, but don't have the mark.
Since I am here editing the file, I marked those tests as well.
I can drop the commit if that's not helpful.